### DISCUSSION TITLE
Help with subtasks formula

### DISCUSSION THREAD

**Discussion by esantelmo** [thegetty/quire#985] [Get Help]
Hey everyone
I need a formula that retrieves the average of field "Progress" of all subtasks that include the type "Milestone" ("Marco" in portuguese).
"Tipo" (type) is a multiselect field.

This is how far I've gotten (not working though):

(Status = "Concluída" ? 100 :   (ISNOTEMPTY(Override) ? Override :     (ISEMPTY(subtasks.tipo where any = ["Marco"]) ? 0 :       (SUM(subtasks.tipo where any = ["Marco"]).Progresso / COUNT(subtasks.tipo where any = ["Marco"]))    )  ))

As you can see there are other tests before the one I'm addressing but those work fine.

I believe I should be using a mix of "where" and "in" but those don't seem to work together.

Any help is appreciated!

**Comment by Erin-Cecele** [comment]
Hello @esantelmo! Is this inquiry meant for Quire, the digital publishing tool from Getty: https://quire.getty.edu/?

**Comment by esantelmo** [comment]
oh wow I guess it wasn't lol

Em qui., 19 de dez. de 2024 às 14:08, Erin Cecele Dunigan <
***@***.***> escreveu:
